src/ready/utils/utils.py

```diff
@@ -119,9 +121,118 @@ def sanity_check_trainloader(trainloader, cuda_available):
 
     print(f"############################")
 
-def evaluate_model(model, test_loader, device):
+def create_data_loaders(full_dataset, data_splitting_ratios, seed, batch_size, num_workers):
+    """
+    Create train, validation, and test dataloaders
+    """
+
+    train_set, validation_set, test_set = torch.utils.data.random_split(full_dataset, data_splitting_ratios, torch.Generator().manual_seed(seed)) 
+ 
+    train_loader = torch.utils.data.DataLoader(train_set, batch_size=batch_size, shuffle=True, num_workers=num_workers)
+    validation_loader = torch.utils.data.DataLoader(validation_set, batch_size=batch_size, shuffle=True, num_workers=num_workers) 
+    test_loader = torch.utils.data.DataLoader(test_set, batch_size=batch_size, shuffle=True, num_workers=num_workers)
+
+    return train_loader, validation_loader, test_loader
+
+def training_loop(model, current_idx, current_data, optimizer, training_performance_dict, loss_fn, cuda_available):
+
```

Comment:
Just a minor comment, please add docstrings where missing and move TODO lines inside docstrings